src/app.js

```diff
@@ -0,0 +1,102 @@
+const express =require( "express");
```

Comment:
Codacy has a fix for the issue: [Replace `require(·` with `·require(`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9973834)
```suggestion
const express = require("express");
```